gamestonk_terminal/cryptocurrency/discovery/coinpaprika_model.py

```diff
@@ -0,0 +1,43 @@
+import pandas as pd
+from gamestonk_terminal.cryptocurrency.coinpaprika_helpers import (
+    ENDPOINTS,
+    PaprikaSession,
+)
+
+
+session = PaprikaSession()
+
+
+def search(q, c=None, modifier=None):
+    """Search CoinPaprika
+    Parameters
+    ----------
+    q:  phrase for search
```

Comment:
I would use variables more descriptive. Just chars makes it harder to understand what it is.